newIDE/app/src/VariablesList/VariablesList.js

```diff
@@ -636,7 +636,6 @@ const VariablesList = React.forwardRef<Props, VariablesListInterface>(
           variableContext = getParentVariableContext(variableContext);
         }
         if (variableContext.variable) {
-          // TODO Add ref to child-variables to allow to focus them.
```

Comment:
Why wasn't it done before? Any technical limitation?

Reply:
No, but the feature came with this PR storing all variable name inputs